loops.js

```diff
@@ -30,11 +45,21 @@
 // .... etc.
 // 15 is oneven
 
+for (let i = 0; i <= 15; i++) {
+    if (i === 0) {
+        console.log(i + " is even");
```

Comment:
Is je (i === 0) nodig? Want 0 % 2 = 0, dat is dan toch ook even...? Hmmm... 
Anders zou je hier kunnen volstaan met een if /else ipv een if/elsif/else denk ik dan...?!